plugin/sampling/strategystore/adaptive/aggregator.go

```diff
@@ -144,3 +144,20 @@ func (a *aggregator) Close() error {
 	a.bgFinished.Wait()
 	return nil
 }
+
+func RecordThroughput(agg strategystore.Aggregator, span *span_model.Span, logger *zap.Logger) {
```

Comment:
My initial plan was to refactor `cmd/collector/app/root_span_handler.go`. but, `handleRootSpan` is tightly coupled with the collector code. removing it may cause many irrelevant changes. Therefore, for the time being:
- keep `cmd/collector/app/root_span_handler.go` for collector.
- create a same logic function `aggregator.HandleRootSpan` for v2 sampling.

Reply:
> handleRootSpan is tightly coupled with the collector code

I disagree. `handleRootSpan` is a 15-line function that is invoked from collector, there's nothing else in that file. With the bulk of that function moved to aggregator you can just the lambda function in the collector code, e.g.
```
	if c.aggregator != nil {
		additionalProcessors = append(additionalProcessors, func(span *model.Span, tenant string) {
			c.aggregator.HandleRootSpan(span, tenant)
		})
	}
```